crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -344,22 +353,22 @@ int crypto_sign_open(uint8_t *m,
 {
   size_t i;
 
-  if(smlen < CRYPTO_BYTES)
+  if(smlen < params->bytes)
     goto badsig;
 
-  *mlen = smlen - CRYPTO_BYTES;
-  if(crypto_sign_verify(sm, CRYPTO_BYTES, sm + CRYPTO_BYTES, *mlen, ctx, ctxlen, pk))
+  *mlen = smlen - params->bytes;
+  if(crypto_sign_verify(params,sm, params->bytes, sm + params->bytes, *mlen, ctx, ctxlen, pk))
     goto badsig;
   else {
     /* All good, copy msg, return 0 */
     for(i = 0; i < *mlen; ++i)
-      m[i] = sm[CRYPTO_BYTES + i];
+      m[i] = sm[params->bytes + i];
     return 0;
   }
 
-badsig:
-  /* Signature verification failed */
-  *mlen = 0;
+  badsig:
+    /* Signature verification failed */
+    *mlen = 0;
   for(i = 0; i < smlen; ++i)
     m[i] = 0;
```

Comment:
Accidental change? Spacing with the for loop is now off.

Reply:
Thank you, fixed in [1d896b1](https://github.com/aws/aws-lc/pull/1910/commits/1d896b1dc79feddf20b3ca4c8bc8ea20e62226db).